packages/toolkit/utils/package.json

```diff
@@ -20,23 +20,28 @@
   "exports": {
     ".": {
       "jsnext:source": "./src/index.ts",
-      "default": "./dist/index.js"
+      "import": "./dist/esm/index.js",
+      "default": "./dist/cjs/index.js"
     },
     "./ssr": {
       "jsnext:source": "./src/ssr.ts",
-      "default": "./dist/ssr.js"
+      "import": "./dist/esm/ssr.js",
+      "default": "./dist/cjs/ssr.js"
     },
     "./logger": {
       "jsnext:source": "./src/logger.ts",
-      "default": "./dist/logger.js"
+      "import": "./dist/esm/logger.js",
+      "default": "./dist/cjs/logger.js"
     },
     "./constants": {
       "jsnext:source": "./src/constants.ts",
-      "default": "./dist/constants.js"
+      "import": "./dist/esm/constants.js",
+      "default": "./dist/cjs/constants.js"
     },
     "./babel": {
       "jsnext:source": "./src/babel.ts",
-      "default": "./dist/babel.js"
+      "import": "./dist/esm/babel.js",
+      "default": "./dist/cjs/babel.js"
     },
     "./ajv": "./compiled/ajv/index.js",
```

Comment:
The `compiled` folder should be added to `.npmignore`, and `./compiled` can be replaced by `./dist/compiled`.

So we do not need to publish two duplicated `compiled` folder.

Reply:
done

Reply:
The exports path should be replaced too, such as `./compiled/ajv/index.js`